m4_sessions/1806-inning/group2.md

```diff
@@ -1,33 +1,55 @@
 ## Group Members:
+- Kolya Venturi
+- Jake Rambold
+- Stephanie Ospina
 
 
-
-## Name of Session: 
+## Name of Session: Escaping Your Echo Chamber
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Living in an increasingly polarized society, it has become much harder to understand what has been going on outside of your own bubble, let alone have a conversation with people with opposing ideas. By understanding your echo chamber, you can better understand the people around you and in the process expand your world views.
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+- Better understanding of their cognative biases
+- An understanding of the backfire effect when viewing opposing content
+- An understanding of how social media actively encourages echo chambers
 
 ## Session Outline 
-
-(Please include who is facilitating and how much time you are going to spend in each section):
-
-
-
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
+    - Intros (All)
+    - What is an echo chamber? (Kolya)
+    - The motivation behind them (Jake)
+      - Why do people fall into them
+      - What is the reason they stay in them
+    - Confliciting moral values (Stephanie)
+    - Discuss: Where do you see echo chambers in your life? (Kolya)
+    - Short video on cognative dissonance 
+    - Short discussion on video (Jake)
+    - 5-ish minutes of reading
+      - Oatmeal comic (backfire effect)
+      - Maybe an article about something(?)
+    - How can certain arguments shut down discussion (Stephanie)
+      - People argue for the sake of arguing, not always to solve a problem
+    - Activity about arguing the other side (All)
+    - Closing (All)
```

Comment:
Be a bit more specific--how can students leave with "something" or new understanding?  What reflection do you want them to have by the end of the session?